### PR TITLE
Adds a post about testing private methods.

### PR DESCRIPTION
**Pull request by michaeldupuisjr**

### CODE REVIEW

File: president-guesser.rb [hunk 1]
```diff
@@ -0,0 +1,84 @@
+class President
```
Comment (bcardarella) [inline president-guesser.rb]: why is this file here?

**Comment by michaeldupuisjr** [comment]
@bcardarella I had been using that file to copy and paste the example code and I committed it by mistake. It has been removed.

**Comment by bcardarella** [comment]
@michaeldupuisjr you could modify this post with elixir as the language and perspective instead of ruby

**Comment by michaeldupuisjr** [comment]
@bcardarella I could, but one of the strongest arguments for not testing private methods is that doing so breaks encapsulation. I don't think this is a concern in non-OOP, so I'm not sure how I could apply it to a functional language. I'll think on it though, and see if I can come up with anything.

**Comment by bcardarella** [comment]
Now that I think of it, I don't even believe you can test private functions in Elixir. There is no way to access the function outside of the module.
